cfpb/sbl-frontend
# feat: [Filing] Verify user is associated with the LEI they are trying to access

Close #720 

Builds upon Filing route protections

## Changes

- [Filing]: Verify user is associated with the LEI they're trying to access
- [Institution] Verify user is associated with the LEI they're trying to access

## How to test
- `yarn seed`
- Associate your account with only one of the seed institutions
- Attempt to access Filing (upload, submit, etc) or Institution (view, update, etc) pages for an institution you are **NOT** associated with (put LEI in the URL)
- Confirm that you are redirected to the Platform home page 

## Screenshots
Happy to demo, msg me
